resources/.ktlint_editorconfig

```diff
@@ -4,3 +4,11 @@ ktlint_standard_max-line-length = 160
 ktlint_standard = enabled
 ktlint_standard_multiline-expression-wrapping = disabled
 ktlint_standard_string-template-indent = disabled
+ktlint_standard_chain-method-continuation = disabled
+ktlint_standard_class-signature = disabled
+ktlint_standard_trailing-comma-on-call-site = disabled
+ktlint_standard_function-signature = disabled
+ktlint_standard_trailing-comma-on-declaration-site = disabled
+ktlint_standard_if-else-wrapping = disabled
+ktlint_standard_multiline-if-else = disabled
```

Comment:
🔧 These are standard [Kotlin conventions](https://kotlinlang.org/docs/coding-conventions.html#control-flow-statements), so I don't think they should be removed.